core-bundle/tests/EventListener/InsertTags/TranslationListenerTest.php

```diff
@@ -59,10 +59,12 @@ public function testIgnoresOtherInsertTags(): void
             ->method('trans')
         ;
 
-        $translator
-            ->expects($this->never())
-            ->method('transChoice')
-        ;
+        if (method_exists($translator, 'transChoice')) {
```

Comment:
Looks like this should use `$translator->expects($this->never())->method($this->any())` ?

Reply:
Changed in 13d8b5b885b4749b1c18174cb566c8d889a2a099.